staging/src/k8s.io/client-go/dynamic/client.go

```diff
@@ -244,18 +246,36 @@ func (rc *ResourceClient) Watch(opts metav1.ListOptions) (watch.Interface, error
 		Watch()
 }
 
-func (rc *ResourceClient) Patch(name string, pt types.PatchType, data []byte) (*unstructured.Unstructured, error) {
+func (rc *ResourceClient) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (*unstructured.Unstructured, error) {
 	result := new(unstructured.Unstructured)
 	err := rc.cl.Patch(pt).
 		NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
 		Resource(rc.resource.Name).
```

Comment:
When a `ResourceClient` gets created from `func (c *Client) Resource(resource *metav1.APIResource, namespace string) ResourceInterface`, can `resource.Name` contain `/`, e.g. `pods/binding`? Currently this line `Resource(rc.resource.Name)` will throw an error "invalid resource may not contain '/'", but I suppose `metav1.APIResource.Name` should allow '/' to reflect what we are serving in api discovery: 
    {
      "name": "pods/binding",
      "singularName": "",
      "namespaced": true,
      "kind": "Binding",
      "verbs": [
        "create"
      ]
    },